Geometry/HcalCommonData/python/hcalSimDBConstants_cff.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+from Geometry.HcalCommonData.hcalSimulationParameters_cff   import *
+from Geometry.HcalCommonData.hcalSimulationConstants_cfi    import *
+from Geometry.HcalCommonData.hcalDBConstants_cff            import *
+from Geometry.HcalCommonData.caloSimulationParameters_cff   import *
+from Geometry.HcalCommonData.hcalDDDSimConstants_cfi        import *
+from Geometry.HcalCommonData.hcalDDDRecConstants_cfi        import *
```

Comment:
Are the hcalDDD*Constants_cfi needed or not? In the other above files you removed them.

Reply:
I think they are